required/tools/tools.ins

```diff
@@ -213,12 +213,10 @@ given in the file `manifest.txt'.
 
 \Msg{}
 \Msg{* Finally trying to make a file `.tex'.}
-\Msg{* (Placed at the end of this run, as this}
-\Msg{*  may fail on some operating systems.)}
+\Msg{* TeX may be blocked from writing .tex so writing rename-to-empty-base.tex}
+\Msg{* Rename that file to .tex after the docstrip generation}
```

Comment:
For consistency with the other `\Msg` lines:

```suggestion
\Msg{* TeX may be blocked from writing `.tex' so writing `rename-to-empty-base.tex'}
\Msg{* Rename that file to `.tex' after the docstrip generation}
```

Also, a few lines further up in this file (seems I can't comment there) these two lines are now obsolete:

```latex
\Msg{* Note there also may be a file .tex which is `invisible'}
\Msg{* on some operating systems.}
```

Reply:
@user227621  yes I handled both in 3e8ee8383